### PR TITLE
CEDS-2416 - Move the Previous Documents Page into the GDS

### PR DESCRIPTION
**Pull request by SteveSugden**

### CODE REVIEW

File: app/views/components/gds/exportsInputText.scala.html [hunk 1]
```diff
@@ -45,5 +45,5 @@
     label = buildLabel,
     hint = hintKey.map(key => Hint(content = Text(messages(key)))),
     errorMessage = field.error.map(err => ErrorMessage(content = Text(messages(err.message)))),
-    classes = defaultInputClasses + inputClasses.map(clazz => s" $clazz").getOrElse("")
+    classes = inputClasses.map(clazz => s" $clazz").getOrElse(defaultInputClasses)
```
Comment (PatrykRudnicki) [inline app/views/components/gds/exportsInputText.scala.html:48]: 👍

File: app/views/declaration/goods_measure.scala.html [hunk 2]
```diff
@@ -1,74 +0,0 @@
...
-
-@this(main_template: views.html.main_template)
-
-@(mode: Mode, itemId: String, form: Form[CommodityMeasure])(implicit request: JourneyRequest[_], messages: Messages)
-
-@tariffBodyDesc = { @Html(messages("declaration.type.consignmentTariffText", components.details_content_link())) }
-
-@tariffBody = { @components.tariff_body_partial(messages("declaration.type.commodityMeasureListItemOne"), messages("declaration.type.commodityMeasureListItemTwo"), messages("declaration.type.commodityMeasureListItemThree"))(tariffBodyDesc)}
-
-@main_template(
```
Comment (PatrykRudnicki) [inline app/views/declaration/goods_measure.scala.html:32]: Is it some unused page?

Comment (PatrykRudnicki) [reply app/views/declaration/goods_measure.scala.html:32]: Probably we should be able in that case clear the messages that are not used after removing the file

Comment (SteveSugden) [reply app/views/declaration/goods_measure.scala.html:32]: Yes, it's not used.  I was looking for pages that were still not using GDS.  Good point about the messages.  I'll do that now.

Comment (PatrykRudnicki) [reply app/views/declaration/goods_measure.scala.html:32]: I should read the commit messages to not ask stupid questions 😂

Comment (SteveSugden) [reply app/views/declaration/goods_measure.scala.html:32]: It's early.  And Friday!

Comment (SteveSugden) [reply app/views/declaration/goods_measure.scala.html:32]: Just checked - there are no messages referenced in this file that aren't either being used elsewhere or not in the messages.en file (i.e. already purged)

**Pull request merged**